README.md

```diff
@@ -42,6 +39,10 @@ Development best practices for Intrepid
     * [Rails gotchas](webservices/gotchas/rails.md)
     * [MEAN stack glossary](webservices/mean_stack_glossary.md)
 
+## Jenkins
+* Configuration for all platforms
+    * [Intrepid Jenkins Configuration](https://source.digital.accenture.com/projects/INT/repos/shhherpa/browse/Jenkins)
```

Comment:
Do we need this link at all in the public repo since it only applies internally?

I know it'd be useful for our devs to have this for reference, but anyone from outside the org will be confused since this will be a dead link for them.

Instead we could maintain a Slack pin or some other record for this link to be easily accessible to internal devs?

Reply:
Fair point -- I added it because I didn't want to just remove the links all together. If everyone agrees that a pin to shhherpa would be sufficient, then that's cool with me